homeassistant/helpers/service.py

```diff
@@ -340,7 +340,13 @@ def async_set_service_schema(
 
 
 @bind_hass
-async def entity_service_call(hass, platforms, func, call, required_features=None):
+async def entity_service_call(
+    hass: HomeAssistantType,
+    platforms: Iterable["EntityPlatform"],
+    func: Union[str, Callable[..., Any]],
+    call: ha.ServiceCall,
+    required_features: Optional[Iterable] = None,
```

Comment:
```suggestion
    required_features: Optional[Iterable[int]] = None,
```

Reply:
Hm, causes an issue with entity.supported_features being Optional elsewhere in the file, but will address.